src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/Et3ResponseHelper.java

```diff
@@ -17,9 +22,14 @@
     "PMD.ExcessiveMethodLength", "PMD.ClassNamingConventions", "PMD.PrematureDeclaration"})
 public class Et3ResponseHelper {
 
+    private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+    private static final String TEMPLATE_NAME = "ET3_0922 (1).docx";
+    private static final String OUTPUT_NAME = "ET3 Response.pdf";
```

Comment:
Just wondering - if we had multiple respondents, would we be better off appending the respondents name to the output name? e.g Jack Reeve - ET3 Response?